toolchain/check/context.h

```diff
@@ -44,7 +44,11 @@ struct LookupResult {
   // was not found in a specific.
   SemIR::SpecificId specific_id;
   // The declaration that was found by name lookup.
+  // Invalid for poisoned items.
+  // TODO: Make this point to the poisoning declaration.
   SemIR::InstId inst_id;
+  // Whether the lookup found a poisoned name.
+  bool is_poisoned = false;
```

Comment:
Is this default value used? I wonder if it might already be getting explicitly set, such that the default could be removed.

Reply:
I think it depends on how the struct is initialized, so this is safer.
If in some function definition you just do
`LookupResult result;`
Then I think nothing guarantees `is_poisoned` would be false.